docs/usage/self-hosted-experimental-flags.md

```diff
@@ -0,0 +1,35 @@
+# Self-hosted experimental flags
+
+The following flags are "experimental" because they:
+
+- are not commonly needed
+- are typically an effort to work around some other service's or platform's problem
+- can be removed at any time
+- are variables for Renovate's internal use to validate they work as intended
+
+Experimental variables which are commonly used and for which there is no external solution in sight can be converted to an official configuration option by the Renovate bot developers.
+
+Use these experimental flags at your own risk.
+We do not follow Semantic Versioning for any experimental variables.
+These flags may be removed or have their behavior changed in **any** version.
+We will try to keep breakage to a minimum, but make no guarantees that an experimental flag will keep working.
+
+## `dockerHubTags`
+
+If added to the `experimentalFlags` list, Renovate will use the Docker Hub API (`https://hub.docker.com`) to fetch tags instead of the normal Docker API for images pulled from `https://index.docker.io`.
+
+```js
+experimentalFlags: ['dockerHubTags'];
```

Comment:
```suggestion
experimentalFlags: ['dockerHubTags=true'];
```

Shouldn't we explicit use bool values? because we will invert the flag in one of the next majors?

Reply:
Okay, will update.